gtk3/lib/gtk3/deprecated.rb

```diff
@@ -6,6 +6,8 @@ module Gtk
   define_deprecated_const :FileSelection,          :raise => "Use 'Gtk::FileChooserDialog' instead."
   define_deprecated_const :FontSelection,          :raise => "Use 'Gtk::FontChooserWidget' instead."
   define_deprecated_const :FontSelectionDialog,    :raise => "Use 'Gtk::FontChooserDialog' instead."
+  define_deprecated_const :ColorSelection          :raise => "Use 'Gtk::ColorChooserWidget' instead."
```

Comment:
Add `remove_const(:ColorSelection)` before the line. `define_deprecated_const` uses `const_missing` hook. So we can't have defined constant.